### DISCUSSION TITLE
Cant get skins to work

### DISCUSSION THREAD

**Discussion by ghost** [LibreSprite/LibreSprite#366] [Q&A]
no matter what I do I cant get any libresprite skins/themes to work :( pls help me

**Comment by maujin111** [comment]
What operating system are you using?

**Reply by ghost** [reply]
Windows 10 pro 64-bit

**Reply by maujin111** [reply]
Delete the folder "Libresprite on " C:\Users<your-username>\AppData\Roaming\ and restart the program
